### PR TITLE
Document the process of porting over samples to hero sample repositories

### PR DESCRIPTION
**Pull request by prprabhu-ms**

### CODE REVIEW

File: docs/releases/updating-hero-samples.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## When to update the hero samples
+
+We typically update hero samples for two distinct reasons:
+
+- To address any critical bugs in the hero sample.
+  - The most common example of this are security issues discovered by [depandabot](https://github.com/dependabot) filed as GitHub issues on the hero sample repositories.
+- To update the hero samples from the corresponding upstream samples.
+  - Hero samples showcase the latest [stable release of `@azure/communication-react`](https://www.npmjs.com/package/@azure/communication-react).
+  - We should update the hero samples from upstream every time we release a new stable version for `@azure/communication-react`, to showcase new features being shipped.
```
Comment (prprabhu-ms) [inline docs/releases/updating-hero-samples.md:36]: Comment on adding references to upstream fix in the hero sample PR.

File: docs/releases/updating-hero-samples.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Updating hero samples
```
Comment (prprabhu-ms) [inline docs/releases/updating-hero-samples.md:1]: Rename file to `hero-samples.md`

Comment (prprabhu-ms) [reply docs/releases/updating-hero-samples.md:1]: ?

File: docs/releases/creating-a-release.md [hunk 3]
```diff
@@ -135,6 +135,12 @@ Even though most of the steps necessary to release the package are automated via
     ![Trigger the deploy storybook action](../images/trigger-storybook-publish.png)
 1. Once everything is deployed and published on npm, delete the release branch from github.
 
+## Step ∞
+
+*For stable release only*, update the hero samples to use the newly released stable package. See [instructions for updating hero samples](./hero-samples.md).
+
+It is acceptable to update the samples within ~a week of the package release.
```
Comment (JamesBurnside) [inline docs/releases/creating-a-release.md:142]: nit: Can this be more assertive: 
```suggestion
Samples should be updated within a week of the package release
```

File: docs/releases/hero-samples.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+Updating hero samples from upstream samples is a manual process. You can minimize the toil by following the process recommended in this section.
+
+This section uses [an update to the chat hero sample](https://github.com/Azure-Samples/communication-services-web-chat-hero/pull/69) as an example. Follow similar steps when updating the other hero samples.
+
+- Pick the git commit in this monorepository to update from.
+  - By default, pick the git tag for the latest stable release. This ensures that the upstream sample is consistent with the released `@azure/communication-react` version.
+  - You may want to pick a later commit to pull in some useful updates to the sample.
+    - If you do, make sure that the intervening commits do not introduce changes that use features or API unavailable in the latest stable release of `@azure/communication-react`.
+  - For this example, I synced from [a541b2](https://github.com/Azure/communication-ui-library/tree/a541b2294943cdd4f885fdfc0ae60511f95c960a/samples/Chat). This was not the latest stable release, but the log of changes since the stable release showed only one very useful change:
+  ```sh
```
Comment (JamesBurnside) [inline docs/releases/hero-samples.md:58]: nit: this code block should be indented another space

File: docs/releases/hero-samples.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+## How To: Update from upstream
+
+Updating hero samples from upstream samples is a manual process. You can minimize the toil by following the process recommended in this section.
+
+This section uses [an update to the chat hero sample](https://github.com/Azure-Samples/communication-services-web-chat-hero/pull/69) as an example. Follow similar steps when updating the other hero samples.
+
+- Pick the git commit in this monorepository to update from.
+  - By default, pick the git tag for the latest stable release. This ensures that the upstream sample is consistent with the released `@azure/communication-react` version.
+  - You may want to pick a later commit to pull in some useful updates to the sample.
```
Comment (JamesBurnside) [inline docs/releases/hero-samples.md:55]: I would suggest to ignore this to keep instructions simple and only have choosing the git tag. We could add a section "out-of-band-hero-sample-updates" where app-only updates in between stable releases can be cherry picked in manually

**Review by JamesBurnside — APPROVED** [review]

**Review by prabhjot-msft — APPROVED** [review]
@prprabhu-ms thanks for creating this doc, is it possible if we can provide link of this doc in the OCE tasks related to hero samples!

File: docs/releases/hero-samples.md (hunk 5 above)
Comment (prprabhu-ms) [reply docs/releases/hero-samples.md:55]: Agreed. Dropped those points.

**Pull request merged**